lib/index.js

```diff
@@ -13,12 +15,20 @@ const linterUiDefault = {
       atom.packages.getLoadedPackage('linter-ui-default').metadata['package-deps'].push('busy-signal')
     }
 
-    if (!atom.inSpecMode()) {
-      // eslint-disable-next-line global-require
-      require('atom-package-deps').install('linter-ui-default', true)
+    let callbackID
+    const installLinterUIDefaultDeps = () => {
```

Comment:
The profiler _only_ shows the file name... not the path (except in rare and random cases). We can have both benefits though, will fix in a few minutes.